### PR TITLE
Migrate netwrok partition test to sig apps

### PR DESCRIPTION
**Pull request by guangxuli**
**What this PR does / why we need it**:

Migrate network partition relevant e2e test to sig-app. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Ref Umbrella issue #49161

**Special notes for your reviewer**:

**Release note**:
```release-note
none
```

### CODE REVIEW

**Comment by guangxuli** [comment]
@liggitt @enisoc PTAL.

File: test/e2e/apps/network_partition.go [hunk 1]
```diff
@@ -43,7 +43,6 @@ import (
 )
 
 const (
-	timeout                = 60 * time.Second
```
Comment (xiangpengzhao) [inline test/e2e/apps/network_partition.go]: BTW, why remove this const here?

Comment (guangxuli) [reply test/e2e/apps/network_partition.go]: because this const has been defined in another file (disruption.go) in this pkg.

Comment (xiangpengzhao) [reply test/e2e/apps/network_partition.go]: Got it, thanks!

**Pull request merged**